sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/package.scala

```diff
@@ -100,6 +70,23 @@ package object util extends Logging {
     file
   }
 
+  private def writeToByteArrayOutputStream(inStream: InputStream): ByteArrayOutputStream = {
```

Comment:
Looks like the new private method is always used with .toByteArray; can it just return the byte array?

Reply:
[0fd84ad](https://github.com/apache/spark/pull/35622/commits/0fd84adee015137fead174122f1891972ba97f5a) rename this method to `toByteArray` and change return type to `Array[Byte]`